content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -201,60 +241,79 @@ git push -u origin main
 
 You're ready to import your project into the Prisma Data Platform.
 
-## 7. Importing your Project into the Prisma Data Platform
+## 8. Import your Project into the Prisma Data Platform
 
 With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with the [Data Proxy](/data-platform/data-proxy).
 
-To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
+1. To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
+   <Admonition type="info">
+   
+   **Note**<br /><br />
+   You only need to have a GitHub account to automatically sign up when you log in for the first time.
```

Comment:
I also need the GH account later. I think you probably want to say something else here.

Reply:
I am not sure I understand this comment.

Reply:
I guess the way it's currently phrased, it sounds confusing. I meant to say that the only requirement is to have an existing GitHub account. Something like this instead:

_Signing up for using the Prisma Data Platform requires that you have a GitHub account. Signup and login are automatic if you are already logged in to GitHub._

Reply:
Ah ok.

> Signup and login are automatic if you are already logged in to GitHub.

I think this part is still confusing. I think you can just leave it off, or rewrite that the PDP account will just be logged use the email address of the GH account but not need other input?

Reply:
I feel like you can leave out the Admonition. The info about having a GitHub account is actually part of the topic about "Using PDP", not the topic of deploying to CF Workers. 

I mean it is polite and nice to make sure people know about needing a GH account to use PDP, but maybe it is too much info for here?

Reply:
If you really want to say something, perhaps simply "A GitHub account is required to sign up for Prisma Data Platform."

Reply:
Indeed, I've kept this last suggested version. Thank you, Tana.